orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/sigs/SignatureAllocation.java

```diff
@@ -16,4 +16,6 @@ public interface SignatureAllocation
     double allocation();
 
     double percent();
+
+    String etiology();
```

Comment:
Should be NotNull :)

Also, I think I would put this property under "signature" (same as what you have done in the example jsons, feels like natural order.